### ISSUE TITLE
Implement install by default full text search

### ISSUE THREAD

**Issue by c0fe** [owncloud/core#22741]
Implement the ability to perform full text search instead of Search Lucene which has been buggy and has not even been released properly.

**Comment by RealRancor** [comment]
Maybe just get https://github.com/owncloud/search_lucene/issues/113 fixed for 8.2?

**Comment by c0fe** [comment]
Search Lucene is still not enough, there are also various bug issues with it.

**Comment by RealRancor** [comment]
Not everyone wants to have fulltext search on as a default. Thats the main advantage of ownCloud, you can have stuff like this in apps.

So why do you think its better to include the functionality into ownCloud instead of fixing your mentioned bugs in search lucene?

**Comment by c0fe** [comment]
RealRancor, this can be implemented as an option that can be enabled or disabled. I would recommend keeping it disabled on install or offering it as an option during initial installation. The point is this is an important feature and leaving it as an app would mean it wouldn't function as well.

**Comment by stuba** [comment]
We are testing OC for one of our customers and I find it scary that this feature just can drop off. Never to mention about System Administrator manual for 8.2, that just says "Using the Full-Text Search app is literally set-it-and-forget-it: all you do is enable it on your Apps page"..

Does this effect on Enterprise users as well?

**Comment by fwolfst** [comment]
I'd love to see full-text search actually working (does not yet appear to work in 9.x) and would be fine to "opt-in". The documentation is useless to me (opened https://github.com/owncloud/documentation/issues/2372).

**Comment by carlaschroder** [comment]
Where is the Search app? It is not present in 8.2 or 9.0. Not sure who to ask-- @butonic @icewind1991 @jnweiger

**Comment by carlaschroder** [comment]
Meh. Looks like it's gone from 8.2 and 9.0. You need to tell me these things, friends!

**Comment by carlaschroder** [comment]
Doc pages removed https://github.com/owncloud/documentation/issues/2372#issuecomment-212981024

**Comment by Xenopathic** [comment]
@carlaschroder It's not been removed per-se, it's just available as a non-default app on the appstore. Not sure why that decision was made though

**Comment by carlaschroder** [comment]
@Xenopathic See https://github.com/owncloud/search_lucene/issues/113
tl;dr: not compatible with 8.2+. And it's buggy.

**Comment by Xenopathic** [comment]
According to https://github.com/owncloud/search_lucene/releases there have been more recent releases?

**Comment by LukasReschke** [comment]
The tags are a lie. They are automatically tagged since it's in the release script. However, that won't magically make it compatible 🙈

**Comment by fwolfst** [comment]
Any light on the end of the tunnel? Would be such a lovely feature. And I'd reckon its a pretty enterpricy one, too.

**Comment by hodyroff** [comment]
Full Text Search is available as a Consulting Project for our customers, indeed. Integration is offered with ElasticSearch, but feature is by far not complete or ready for all use cases. In any case its a seperate app, not part of core. Happy for community apps.
